javaslang-benchmark/src/test/java/javaslang/JmhRunner.java

```diff
@@ -91,4 +111,8 @@ private static void runAndReport(Class<?> benchmarkClass, int warmupIterations,
         }
         return results;
     }
+
+    public static int xor(int x, int y) {
```

Comment:
I would write `x ^ y` instead of `xor(x, y)` - but its just the src/test package, so leave it as is.